examples/models/twin_axis.py

```diff
@@ -1,3 +1,12 @@
+''' This example demonstrates how to create a plot with two y-axes using Bokeh and NumPy.
```

Comment:
"with Bokeh" is implicity, and numpy usage is really just incidental. Suggest:
```suggestion
''' This example demonstrates how to create a plot with two y-axes.
```

Reply:
Oh, thank you @bryevdv 

I really appreciate the comments and feedback, its a learning curve for me